src/main/resources/TourCode/StepClass.xml

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This page seems exactly the same except that it has been exported differently (the field orders are not the same, but the content is). Could you confirm? If it's right, please remove your change.